exercise-3/flattArray.js

```diff
@@ -0,0 +1,20 @@
+function flatArray(array){
+    return array.flat(nestingDepth(array))
+}
+function nestingDepth(array){
+    if(!array.length) return 0
+    let arrStr = JSON.stringify(array)
+    const arrStrLen  = arrStr.length, depths =[]
+    // Don not count in the outside [] of an array
+    arrStr = arrStr.slice(1,arrStrLen-1)
+    let i=0
+    while(i<arrStrLen-2){
```

Comment:
I think you're using `arrStrLen - 2` in while loop because you removed the the outer parenthesis after getting the length of the string

Reply:
Would you recomend having another variable to keep new length?